instrumentation/aws_lambda/lib/opentelemetry/instrumentation/aws_lambda/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsLambda
+      VERSION = '0.0.1'
```

Comment:
```suggestion
      VERSION = '0.1.0'
```
Since the third column is for patches, I think moving the one to the second/minor column may make more sense.